managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ReadOnlyCursor.java

```diff
@@ -22,9 +22,13 @@
 
 import com.google.common.base.Predicate;
 import com.google.common.collect.Range;
+import org.apache.bookkeeper.common.annotation.InterfaceAudience;
```

Comment:
why are you using "org.apache.bookkeeper.common" and not "org.apache.pulsar.common" here and in other files  ?
it is a typo? or you did it for some reason ?

Reply:
managed-ledger already used bookkeeper annotations. I am keeping it consistent.

Reply:
ok